docs/cudf/source/basics/basics.rst

```diff
@@ -15,32 +15,61 @@ The following table lists all of cudf types. For methods requiring dtype argumen
+    | Datetime               |                              | np.datetime64_                                                                      | ``'datetime64[s]'``, ``'datetime64[ms]'``,  |
+    |                        |                              |                                                                                     | ``'datetime64[us]'``, ``'datetime64[ns]'``  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Timedelta              |                              | np.timedelta64_                                                                     | ``'timedelta64[s]'``, ``'timedelta64[ms]'``,|
+    | (duration type)        |                              |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Categorical            | CategoricalDtype             | (none)                                                                              | ``'category'``                              |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Boolean                |                              | np.bool_                                                                            | ``'bool'``                                  |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Decimal                | Decimal64Dtype               | (none)                                                                              | (none)                                      |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | List                   | ListDtype₁.₁                 | np.list_                                                                            | (none):sub:``1.2``                          |
+    +------------------------+------------------------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+    | Struct                 | StructDtype₁.₁               | np.struct_                                                                          | (none):sub:``1.2``                          |
```

Comment:
Numpy has structured arrays, but perhaps Spark is the right analog here despite that as our functionality is more aligned to what's supported in big data processing frameworks? The dtype isn't `np.struct_` as the base dtype for a numpy structured array is void I think.

The link may also be redirecting to itself.

In the "View File" version, the subscript on `(none)` is also not behaving well.

Reply:
Also, There isn't a `np.list`/`np.struct` class because numpy does not support nested types (excluding multidimensional arrays). The corresponding host scalar type for these are plain python list and dict. Analogous to how `str` was done above, you should put in links to these `stdtypes`:

list: https://docs.python.org/3/library/stdtypes.html#list
dict: https://docs.python.org/3/library/stdtypes.html#dict

Reply:
Agreed, will be resolved in the next commit.